App10.1/App10.1/10(1).cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+internal class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите число числа a и b, при этом число a должно быть меньше числа b");
+
+        Console.WriteLine("Число a = ");
+        int a = int.Parse(Console.ReadLine());
+
+        Console.WriteLine("Число b = ");
+        int b = int.Parse(Console.ReadLine());
+
+        if (a <= 0 || b <= a)
+        {
+            Console.WriteLine("Должно выполняться условие 0 < a < b");
+            return;
+        }
+
+        double sumOfCubes = 0;
+        for (int i = a; i <= b; i++)
+        {
+            sumOfCubes += Math.Pow(i, 3);
+        }
```

Comment:
Сумма кубов целых чисел - целое число. Формат с плавающей точкой тут неуместен.
Если в теле цикла один оператор, скобки можно не писать.
```suggestion
        int sumOfCubes = 0;
        for (int i = a; i <= b; i++)
            sumOfCubes += (int)Math.Pow(i, 3);
```
или
```suggestion
        int sumOfCubes = 0;
        for (int i = a; i <= b; i++)
            sumOfCubes += i * i * i;
```